apps/wallet-mobile/src/yoroi-wallets/cardano/getMinAmounts.ts

```diff
@@ -36,16 +36,16 @@ export const getMinAmounts = async (address: Address, amounts: Balance.Amounts,
     {defaultNetworkId: primaryToken.networkId, defaultIdentifier: primaryToken.identifier},
   )
 
-  const [value, coinsPerUtxoByte, bech32Aaddress] = await Promise.all([
+  const [value, coinsPerUtxoByte, address] = await Promise.all([
     cardanoValueFromMultiToken(multiToken),
     CardanoMobile.BigNum.fromStr(COINS_PER_UTXO_BYTE),
-    (await CardanoMobile.ByronAddress.isValid(address))
-      ? await (await CardanoMobile.ByronAddress.fromBase58(address)).toAddress()
-      : await CardanoMobile.Address.fromBech32(address),
+    (await CardanoMobile.ByronAddress.isValid(_address))
```

Comment:
@bankless there is a `normalizeToAddress` function in the `utils` you can use it from there, it has a logic there, yet it also requires tests.

Reply:
i will take a look thanks

Reply:
100% covered

<img width="1384" alt="Screenshot 2023-08-22 at 00 30 01" src="https://github.com/Emurgo/yoroi/assets/105349292/c2846d3c-642a-4cee-aa28-34aefbc01ec3">

Reply:
100% covered

<img width="1384" alt="Screenshot 2023-08-22 at 00 30 01" src="https://github.com/Emurgo/yoroi/assets/105349292/3572a861-13d9-4b7b-8b07-16bea23b9e1a">